components/automate-ui/src/app/components/admin-sidebar/admin-sidebar.component.html

```diff
@@ -26,8 +26,8 @@
       <chef-sidebar-entry route="/settings/tokens" icon="vpn_key">API Tokens</chef-sidebar-entry>
     </app-authorized>
 
-    <div *ngIf="(iamMajorVersion$ | async) !== 'v1'">
-      <div class="group" *ngIf="policies.visible || roles.visible || projects.visible">Access Management</div>
+    <div *ngIf="(iamMajorVersion$ | async) === 'v2'">
+      <div class="group" *ngIf="policies.visible || roles.visible || projects.visible && (iamMinorVersion$ | async) === 'v1'">Access Management</div>
```

Comment:
Seemed like this was being slightly redundant putting it here--why not just move the one on the chef-sidebar up to its parent app-authorized?--but as we discovered, doing that, and changing this to `projects?.visible` failed! Seems like the Angular reference variable (`projects`) is not quite like a real variable after all, so this is a good solution!